core/persistence-jpa/src/test/resources/domains/MasterContent.xml

```diff
@@ -68,9 +68,10 @@ under the License.
   <AuthModule id="4c3ed4e6-7008-11ea-bc55-0242ac130003" name="DefaultGoogleMfaAuthModule"
```

Comment:
You need to apply the same changes in `core/persistence-jpa-json/src/test/resources/domains/MasterContent.xml`